### PR TITLE
Added account deletion capability

### PR DESCRIPTION
**Pull request by NickEckert**
- Added account deletion capability
- Includes warning and confirmation alerts

### CODE REVIEW

File: Deletable.swift [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+//  Created by Nick Eckert on 10/4/17.
+//  Copyright © 2017 sanction. All rights reserved.
+//
+
+import Foundation
+import Alamofire
+import ObjectMapper
+import AlamofireObjectMapper
+
+// The fetchable protocol will allow an object to be capable of fetching data from a URL
```
Comment (RohanNagar) [inline Deletable.swift]: Wrong comment

File: pilot/Fetchable.swift [hunk 2]
```diff
@@ -10,7 +10,6 @@ import Foundation
 import Alamofire
 import ObjectMapper
 import AlamofireObjectMapper
-import SwiftyJSON
```
Comment (RohanNagar) [inline pilot/Fetchable.swift:13]: Good removing unused imports!

File: pilot/ProfileViewController.swift [hunk 3]
```diff
@@ -51,5 +51,44 @@ class ProfileViewController: UITableViewController {
...
+                                                      This action cannot be undone.
+                                                     """,
+                                            preferredStyle: UIAlertControllerStyle.actionSheet)
+        
+        actionSheet.addAction(UIAlertAction(title: "Delete", style: UIAlertActionStyle.default, handler: { _ in
+          
+          UserManager.sharedInstance?.deleteUser(onSuccess: { _ in
+            
+            // Alert the user that the account was succesfully deleted
+            let alert = UIAlertController(title: "User account succesfully deleted!",
```
Comment (RohanNagar) [inline pilot/ProfileViewController.swift]: Typo: succesfully -> successfully

File: Deletable.swift (hunk 1 above)
Comment (NickEckert) [reply Deletable.swift]: You can see how similar fetchable is to deletable.

**Review by RohanNagar — APPROVED** [review]
Great!

**Pull request merged**